presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -1153,6 +1153,20 @@ public void testWrittenIntemediateByteLimit()
         assertQueryFails(session, testQuery, "Query has exceeded WrittenIntermediate Limit of 0MB.*");
     }
 
+    @Test
+    public void testNesteCteWithSameName()
```

Comment:
testNestedCte

Reply:
This test is not only nested CTE, but also CTE with same name in different scope which caused the bug here, hence added same name here in the test name.

Reply:
I think he is referring to the typo in the word **Nested***